src/java/org/apache/cassandra/tcm/AbstractLocalProcessor.java

```diff
@@ -56,7 +56,7 @@ public final Commit.Result commit(Entry.Id entryId, Transformation transform, fi
         {
             ClusterMetadata previous = log.waitForHighestConsecutive();
             if (!previous.fullCMSMembers().contains(FBUtilities.getBroadcastAddressAndPort()))
-                throw new IllegalStateException("Node is not a member of CMS anymore");
+                throw new IllegalStateException("Node " + FBUtilities.getBroadcastAddressAndPort() + " is not a member of CMS anymore in " + previous.epoch + "; members=" + previous.fullCMSMembers());
```

Comment:
this was super flakey and added a ignore filter so the test won't fail if this is seen anymore (assuming its safe... think it is)...

I saw a history where node1 starts a snapshot, node1 looses its CMS membership (nodes 2/3/4 are now the members), then this check blows up...

Reply:
I am going to drop as Marcus is making similar change in his patch....

Reply:
https://issues.apache.org/jira/browse/CASSANDRA-19872